kie-osgi/kie-karaf-itests/src/test/java/org/kie/karaf/itest/DecisionTablesIntegrationTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.karaf.itest;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kie.api.KieBase;
+import org.kie.api.KieServices;
+import org.kie.api.builder.Results;
+import org.kie.api.io.KieResources;
+import org.kie.api.runtime.KieContainer;
+import org.kie.internal.utils.KieHelper;
+import org.ops4j.pax.exam.Configuration;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.karaf.options.LogLevelOption;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+import org.ops4j.pax.exam.spi.reactors.PerMethod;
+
+import static org.junit.Assert.assertTrue;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.configureConsole;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.keepRuntimeFolder;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.logLevel;
+
+import org.drools.decisiontable.ExternalSpreadsheetCompiler;
+
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+public class DecisionTablesIntegrationTest extends AbstractKarafIntegrationTest {
+
+    private static final String SIMPLE_XLS = "/decisiontables/data/MultiSheetDST.xls";
+    private static final String SIMPLE_TEMPLATE = "/decisiontables/templates/test_template1.drl";
+
+    private final KieServices ks = KieServices.Factory.get();
+    private final KieResources kieResources = ks.getResources();
+
+    private ClassLoader origTCCL;
+
+    @Before
+    public void setTCCL() {
+        origTCCL = Thread.currentThread().getContextClassLoader();
+        // Pax-exam sets the TCCL to the bundle-under-test classloader which in turn means that the XStream marshalling
+        // will work with different TCCL than it would with standalone bundle. Setting system/application classloader as
+        // TCCL is needed to reproduce the fails related to XStream unmarshalling in guided-dtables module
```

Comment:
Actually, reading the comment again, removing it will likely not cause the test to fail (which is confusing). In any case, at least the comment should be fixed.